client/src/Components/CourseCheckbox.js

```diff
@@ -1,20 +1,28 @@
-import React from "react";
+import React, { Component } from "react";
 import Checkbox from "./Checkbox";
 
 
-const CourseCheckbox = ({value, onChange }) => {
-    return (
-        <div> Choose the Classes you are currently enrolled in:
-            <Checkbox
-                label = "CSC 001"
-                onChange = {onChange}
-            />
-            <Checkbox
-                label = "CSC 002"
-                onChange = {onChange}
-            />
-        </div>
-    );
-  };
+class CourseCheckbox extends Component {
```

Comment:
Class for selecting courses for students.

Reply:
I would suggest changing the component to a functional component instead of class component. Functional components are easier to manage and have support for React hooks. You also don't need to bind the `this` object.